fineract-provider/src/main/java/org/apache/fineract/portfolio/meeting/service/MeetingReadPlatformServiceImpl.java

```diff
@@ -27,18 +27,14 @@
 import org.apache.fineract.portfolio.calendar.service.CalendarUtils;
 import org.apache.fineract.portfolio.meeting.data.MeetingData;
 import org.apache.fineract.portfolio.meeting.exception.MeetingNotFoundException;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.jdbc.core.RowMapper;
-import org.springframework.stereotype.Service;
 
-@Service
 public class MeetingReadPlatformServiceImpl implements MeetingReadPlatformService {
 
     private final JdbcTemplate jdbcTemplate;
 
-    @Autowired
     public MeetingReadPlatformServiceImpl(final JdbcTemplate jdbcTemplate) {
```

Comment:
Just a minor thing: let's take the opportunity to clean up situations like these... not necessarily wrong, but we can use the Lombok `@RequiredArgsConstructor` class annotation and get rid of the constructor boilerplate code. Here it's not much, but we have other cases where we have a lot of injected dependencies.

Reply:
Oh definitely. I've made the change now